data.py

```diff
@@ -13,11 +16,44 @@ def load() -> tuple[dict[str, dict], dict[str, str]]:
 	text: dict[str, str] = {}
 	with open('data/aura_skill.json', 'rb') as f:
 		raw_text: list[dict] = json.load(f)
-	prefixes = ['You and nearby', 'Aura grants', 'Buff grants']
+	prefixes = ['You and nearby', 'Your and nearby', 'Aura grants', 'Buff grants']
 	for translation in raw_text:
 		for k in translation['ids']:
 			translated = translation['English'][0]
 			if any(translated['string'].startswith(prefix + ' ') for prefix in prefixes):
 				text[k] = translated
 
 	return gems, text
+
+def legion_passive_mapping() -> dict:
+	""" Maps names of timeless legion passives to their effects """
+	# I couldn't find any of this info in the RePoE data, so I'm grabbing it from the path of building repo
+	with open('data/LegionPassives.lua', 'r') as file:
+		content = file.read()
+		content = re.sub(r'\[(\d+)\] =', r'["\1"] =', content)  # turns integer keys into strings
+		content = re.sub(r',\s+}', r'}', content)  # removes commas after the last key value pairs
+		content = re.sub(r'\[(.*)\] =', r'\1:', content)  # removes brackets around keys and changes " =" to ":"
+		content_dict = json.loads(content[content.find('{'):])
+		return {node['dn']: node['sd'].values() for node in content_dict['nodes'].values()}
+
+def militant_faith_node_mapping(seed: int) -> dict:
+	""" Maps passives nodes to their alternative nodes under militant faith"""
+	with zipfile.ZipFile(r'data\MilitantFaithSeeds.zip') as zf:
+		with zf.open('MilitantFaithSeeds.csv', 'r') as infile:
+			reader = csv.reader(io.TextIOWrapper(infile, 'utf-8'))
+			originals = next(reader)[2:]
+			for _ in range(seed - 1998):
+				next(reader)
+			alternatives = next(reader)[2:]
+	return {original: alternative for original, alternative in zip(originals, alternatives)}
+
+def elegant_hubris_node_mapping(seed: int) -> dict:
+	""" Maps passives nodes to their alternative nodes under elegant hubris"""
+	with zipfile.ZipFile(r'data\ElegantHubrisSeeds.zip') as zf:
+		with zf.open('ElegantHubrisSeeds.csv', 'r') as infile:
+			reader = csv.reader(io.TextIOWrapper(infile, 'utf-8'))
+			originals = next(reader)[2:]
+			for _ in range(int(seed / 20) - 99):
```

Comment:
```suggestion
			for _ in range(seed // 20 - 99):
```
any comment you can add about this and the 1998 above would be helpful 'cause I have no idea what's going on here

Reply:
first time i'm ever seeing the double slash, nice!
the csv files have the original names of the notables in the first row and subsequent rows have the alternative names when conquered by timeless jewels with different seeds.
The 1998 comes from militant faith seeds begin at 2000 and the first seed in the csv file being in row 2.
Same principle for the elegant hubris, just that seeds don't increase in increments of 20 rather than 1.

Reply:
should be more understandable now. 
I was kind of anxious interacting too much with the csv since its like 70mb unzipped, but this doesn't reduce the performance